katalog/pomerium/README.md

```diff
@@ -0,0 +1,82 @@
+# Pomerium Setup
+
+This document is intended to give a brief overview on how Pomerium can be implemented, for further details, please look at the official doc : <https://www.pomerium.io/docs/>
```

Comment:
Can we please add a minimal description of what Promerium is? a one-liner should be enough, like we do for the other packages.

Please add it also to the repo's root README packages list :)